v2/support/secure_settings_sample_app/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Bag End Catering Helper",
+  "author": {
+    "name": "Frodo Baggins",
+    "email": "cgoddard@zendesk.com",
+    "url": "https://hobbiton.zendesk-staging.com"
```

Comment:
You probably don't want to give out your email and this URL here. Maybe can use a generic one like [this](https://github.com/zendesk/demo_apps/blob/master/v2/support/basic_ticket_sample_app/manifest.json) instead.